### ISSUE TITLE
Repeated kills are still counted for certain Shadowlands rares while the daily loot lockout quest is already completed (?)

### ISSUE THREAD

**Issue by Duckwhale** [WowRarity/Rarity#374]
Source: [Discord](https://discord.com/channels/788119147740790854/788119314909626388/861730251168415775)
> Can we make 
Rampaging Mauler
Crimson Shardhide
Garnet Razorwing 
Daily kills so they dont keep count

> Can confirm that I've killed Reliwik the Defiant (drops Garnet Razorwing) a second time on the same character on the same lockout and Rarity did count another attempt which I had to manually remove. Not sure about Malbog or Konthrogoz.

> So they can be looted multiple times, but still have a daily loot lockout? That's odd, as normally you can't do this. Rarity tracks when you loot an NPC, so it'll add multiple attempts only if you can actually loot it multiple times. The loot lockout is a serverside flag that tells the server to not generate loot if it's set, as far as I can tell, and has been used for many expansions. Maybe they changed this for Shadowlands? I remember there was the Theater of Pain rares that could be looted multiple times, despite having a lockout, but it was quickly hotfixed when people started abusing it

> I noticed Reliwik did not have a silver portrait the second time I killed him on the same character/same lockout. Probably lootable multiple times per day but only eligible for the mount once per daily lockout. Kroke the Tormented is another rare I've killed several times a day in Korthia that is always lootable (also loses silver portrait after first kill).

> I think cause they made some rare still able to drop meat or trash after they die

This might be related to the Theater of Pain rares, that could be repeatedly looted before a Blizzard hotfix. If loot lockouts don't actually prevent ALL loot, then we need to work around this and account for the defeated questID manually instead of relying on Blizzard to sort this out serverside.

In other words: The assumption "when loot-locked, no drops are obtained" is something we rely on heavily with the ``NPC`` method. This needs to be investigated, and we should first determine if this applies to all Shadowlands rares, legacy rares, or if there's something else that could explain this.
